### PR TITLE
Add backports.tempfile

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: recipes/backports.tempfile/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - python
+    - backports.weakref
+
+test:
+  imports:
+    - backports.tempfile
+
+about:
+  home: https://github.com/pjdelport/backports.tempfile
+  license: PSF
```
Comment (jakirkham) [inline recipes/backports.tempfile/meta.yaml]: Please add version (presumably 2). Also please ask upstream to package the license file as well.

File: recipes/backports.tempfile/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - python
+    - backports.weakref
+
+test:
+  imports:
+    - backports.tempfile
+
+about:
+  home: https://github.com/pjdelport/backports.tempfile
+  license: PSF 2
```
Comment (jakirkham) [inline recipes/backports.tempfile/meta.yaml]: Ideally we would package the license file as well. However one is not currently included in the sdist. Have raised PR ( https://github.com/pjdelport/backports.tempfile/pull/5 ) upstream to do this.

Comment (jakirkham) [reply recipes/backports.tempfile/meta.yaml]: Seems the `sdist` does have the license file after all. Must have missed it before. Have pushed a commit to add it to this PR. Hope that is ok.

Comment (mvdbeek) [reply recipes/backports.tempfile/meta.yaml]: Sure, thanks for that! I didn't see the file either.

**Comment by jakirkham** [comment]
Merging without AppVeyor as this is `noarch`.

**Comment by jakirkham** [comment]
Thanks @mvdbeek. It's in. :)

**Pull request merged**